covid19/src/components/DateErrorMessage.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import { Alert } from '@material-ui/lab';
+
+export default class DateErrorMessage extends Component{
+
+    render() {
```

Comment:
component does not hold any state nor has a lifecycle -> it can then be transformed into a functional component